static/src/javascripts/projects/commercial/modules/messenger/passback.ts

```diff
@@ -51,28 +51,26 @@ const init = (register: RegisterListener): void => {
 			if (!slotId) {
 				log(
 					'commercial',
-					'Passback listener: cannot determine the calling iFrame',
+					'Passback: cannot determine the calling iFrame',
 				);
 			}
 
+			log(
+				'commercial',
+				`Passback: from ${source} for slot ${String(slotId)}`,
+			);
```

Comment:
Does it need to call `String(slotId)` because `slotId` can be undefined? If so, wouldn't we get two sets of log messages

```
Passback: cannot determine the calling iFrame
Passback: from ${source} for slot undefined
```

Does it make sense to do an early return after L54? Or could we wrap this logging in an if-else with the above?

Reply:
It's a very good point. 

I did consider early returns but wanted to get the bug fixes to Ozone asap so if okay I will pick this up as a refactor once they've verified the main flow.

Reply:
Yeah that's cool, this is just the logging so fine either way 🙂